### PR TITLE
Batch Norm

### PR DESCRIPTION
**Pull request by lemairecarl**
My first paper summary. 

Here is a pdf for those who can't run the site locally : [batch-norm-preview.pdf](https://github.com/vitalab/vitalab.github.io/files/765005/batch-norm-preview.pdf)

Any comments are welcome, since we will discover what kind of format we want for our summaries.

Don't be afraid to _**BASH**_. :)

### CODE REVIEW

File: deep-learning/_posts/2017-02-09-batch-norm.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+Notice that if the parameters would learn the values $$ \gamma^{(k)} = \sqrt{\mathrm{Var}[x^{(k)}]} $$ and $$ \beta^{(k)} = \mathrm{E}[x^{(k)}] $$, we would recover the original activations, if that were the optimal thing to do.
+
+**Constant normalization during inference.**{: .phead} During inference, we want the output to depend only on its input (not on other examples). For this, instead of the mini-batch statistics, we use statistics from the whole population, which are constant. In practice, we compute these statistics during training using moving averages. Since the normalization is now fixed, it can be composed with the scaling by $$ \gamma $$ and shift by $$ \beta $$, to yield a single linear transform.
+
+**Why do we need to backprop through it ?**{: .phead} Consider a layer $$ x = u + b $$, where $$ b $$ is a learned bias. We normalize the result using the mean of the activation computed over the whole training data : $$ \hat{x} = x - \mathrm{E}[x] $$. If a gradient descent step ignores the dependence of $$ \mathrm{E}[x] $$ on $$ b $$, then it will update $$ b \leftarrow b + \Delta b $$, where $$ \Delta b \propto \partial \text{loss} / \partial \hat{x} $$. Then,
```
Comment (Dref360) [inline deep-learning/_posts/2017-02-09-batch-norm.md]: $$ \Delta b \propto \partial \text{loss} / \partial \hat{x}

Not shown properly also loss is a function which takes x (therefore the derivative on x)

**Comment by jodoin** [comment]
change title "Batch Normalization" to something more specific.  Maybe the title "Batch normalization: Accelerating deep network training by reducing internal covariate shift"

**Comment by lemairecarl** [comment]
I changed the layout a bit to better accommodate the long title and the full reference. I removed the arXiv reference, since we have already the full ref and we have the pdf.
